package.json

```diff
@@ -186,6 +186,10 @@
           ],
           "markdownDescription": "Optional list of properties to pass along to the Metals server. By default, the environment variable `JAVA_OPTS` and `.jvmopts` file are respected. Each property needs to be a separate item.\n\nExample: `-Dhttps.proxyHost=…`, `-Dhttps.proxyPort=…` or `-Dmetals.statistics=all`"
         },
+        "metals.coursierMirror": {
+          "type": "string",
+          "markdownDescription": "Optional alternative repository to maven central if maven central is not available."
+        },
```

Comment:
What is expected to be set here? It's an url or path to file which is described [here](https://get-coursier.io/blog/#mirrors)? Looking at the the coursier [docs](https://github.com/coursier/coursier/blob/master/modules/paths/src/main/java/coursier/paths/Mirror.java#L26) it's the second

Reply:
or no, I have to look at this closer

Reply:
Anyway, I think the description lacks information on what do we mean by `alternative repository`

Reply:
I would add link to the `https://get-coursier.io/blog/#mirrors` and mention that the user has to provide for example only `https://jcenter.bintray.com`

Reply:
:+1: sure!

Reply:
I would add also `https://get-coursier.io/blog/#mirrors`, wouldn't it be helpful?

Reply:
What if we can add this information "provided link will be used to set `to` property described [here](https://get-coursier.io/blog/#mirrors)"?